modules/blockapis/test/UtxoApi.ts

```diff
@@ -1,3 +1,4 @@
+import 'mocha';
```

Comment:
@byhow used an alternative solution here

it seems like typescript picks up the jasmine definitions from `describe`/`it` sometimes, unless we explicitly import mocha.

Reply:
Thanks for the pointer!